compiler/src/iree/compiler/Codegen/LLVMCPU/test/materialize_x86_64_launch_configuration.mlir

```diff
@@ -1762,12 +1770,15 @@ hal.executable private @reduction_broadcast_pack  {
     }
   }
 }
-//  CHECK-DAG: #[[CONFIG:.+]] = #iree_codegen.lowering_config<tile_sizes = {{\[}}[2, 0], [1, 0], [0, 16], [0, 0]]>
+//  CHECK-DAG: #[[CONFIG1:.+]] = #iree_codegen.lowering_config<tile_sizes = {{\[}}[32, 0], [16, 0], [0, 16], [0, 0]]>
+//  CHECK-DAG: #[[CONFIG2:.+]] = #iree_codegen.lowering_config<tile_sizes = {{\[}}[2, 0], [1, 0], [0, 0], [1, 1]]>
```

Comment:
Not sure I follow how this one is scaled from the pack op to the generic op. Would you mind helping me understand?

Reply:
IIUC it first uses the reduction generic op as the root up to decide the tiling config `[[32, 0], [16, 0], [0, 16], [0, 0]]`.

Then in `setLoweringConfigForComputeOps` we copy and scale the tiling config to other ops. In this case the fill op get `[[32], [16], [0], [0]]` because it only has the first dim, the generic op below get `[[32, 0], [16, 0], [0, 0], [0, 16]]`, which copies the parallel tile sizes from the root op and adds a inner vector tiling `[0, 16]` (following [the logic here](https://github.com/openxla/iree/blob/6710ff6a8f97fd3ee8a455bef8136b4b2ef24009/compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp#L2223))

Then the pack up gets `[[2, 0], [1, 0], [0, 0], [1, 1]]`, which copies and scales down the parallel tile sizes and adds its own inner vector level tiling (follow [the logic here](https://github.com/openxla/iree/blob/6710ff6a8f97fd3ee8a455bef8136b4b2ef24009/compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp#L2221))

Reply:
I added all tiling configs to the test to make it clearer